frontend/packages/dev-console/integration-tests/features/build-config/edit-build-config.feature

```diff
@@ -0,0 +1,94 @@
+@odc-5008
+Feature: Form based edit for Build Configs
+              As a user, I can have form based edit for Build Configs to improve the user experience.
+
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-form-edit-build-config"
+              And user has created a deployment "nodejs-ex-git1"
```

Comment:
This step will repeat for every scenario. So I think you can remove this from the Background and add it to one of the scenario.

Reply:
Updated

Reply:
You have to remove it from the background

Reply:
Removed